tensorboard/plugins/graph/tf_graph_common/render.ts

```diff
@@ -704,7 +704,20 @@ export class RenderGraphInfo {
     // metagraph.
     const originalMetaEdges = this.hierarchy.getPredecessors(
         opNodeToReplace.name);
-    const originalMetaEdge = originalMetaEdges.regular[inputIndex];
+    
+    // Find the metaedge that the input index corresponds to.
+    // A metaedge may correspond to several edges. For instance,
+    // an edge may enter a series node.
+    let originalMetaEdge: Metaedge;
+    let regularEdgeCount = 0;
+    for (let i = 0; i < originalMetaEdges.regular.length; i++) {
+      const metaEdge = originalMetaEdges.regular[i];
```

Comment:
Optional - this could be a forEach, right?  Since you aren't using the index variable except to index into the original list.

Reply:
Yes! Thanks! We now use `_.each`.